src/utils/notification.ts

```diff
@@ -8,21 +8,42 @@ export type DefaultRequestError = {
   status?: number;
 };
 
-export function displayDefaultNotificationFlow<T = unknown>(res?: RTKResponse<T>, success?: () => void) {
-  if (res && 'error' in res && 'data' in res.error) {
-    if (res.error.data) {
+const getErrorFromResponse = (res?: RTKResponse<unknown>) => {
+  if (res && 'error' in res) {
+    let errorObject = null;
+
+    if ('data' in res.error) {
       // @ts-ignore
       const errorTitle = res.error.data?.title || 'Unknown error';
       // @ts-ignore
       const errorDetails = res.error.data?.detail || 'Something went wrong';
 
-      notificationCall('failed', errorTitle, errorDetails);
+      errorObject = {
+        title: errorTitle,
+        message: errorDetails,
+      };
     } else if ('error' in res.error) {
-      notificationCall('failed', res.error.error);
+      errorObject = {
+        // @ts-ignore
+        title: res?.error?.error,
+      };
     }
-  } else {
-    success && success();
+
+    return errorObject;
   }
+};
+
+export function displayDefaultNotificationFlow<T = unknown>(
+  res?: RTKResponse<T>,
+  success?: () => void
```

Comment:
Maybe instead of `success`, it's better to just keep this function `async` (either by `async` keyword, or by returning `Promise.resolve`/`Promise.reject`).

This way it would be covered with `then`/`catch` flow easily

Reply:
Yes, sure. Should I change it on cloud as well?

Reply:
I think so, it will be better to keep it consistent 👍